cli/command/checkpoint/cmd.go

```diff
@@ -16,7 +16,7 @@ func NewCheckpointCommand(dockerCli *command.DockerCli) *cobra.Command {
 			cmd.SetOutput(dockerCli.Err())
 			cmd.HelpFunc()(cmd, args)
 		},
-		Tags: map[string]string{"experimental": "", "version": "1.25"},
+		Tags: map[string]string{"experimental": "", "version": "1.26"},
```

Comment:
@thaJeztah Updated. I was clearly way too over-zealous! What about the `NewVersionError` calls?